api-flask/app/kobo.py

```diff
@@ -122,23 +120,25 @@ def parse_form_response(
     # Some forms do not have geom_field properly setup. So we default to
     # 'geopoint' here and handle edge cases in parse_form_field.
     latlon_dict = parse_form_field(
-        geom_value_string, labels.get(geom_field, "geopoint")
+        geom_value_string, labels.get(geom_field, "geopoint")  # type: ignore
     )
 
-    status = form_dict.get("_validation_status").get("label", None)
-    form_data = {**form_data, **latlon_dict, "date": datetime_value, "status": status}
+    status = form_dict.get("_validation_status").get("label", None)  # type: ignore
+    form_data = {**form_data, **latlon_dict, "date": datetime_value, "status": status}  # type: ignore
 
     logger.debug("Kobo data parsed as: %s", form_data)
 
     return form_data
 
 
-def parse_datetime_params():
+def parse_datetime_params(
+    begin_datetime_str: str, end_datetime_str: str | None = None
+) -> tuple[datetime, datetime]:
     """Transform into datetime objects used for filtering form responses."""
-    begin_datetime_str = request.args.get("beginDateTime", "2000-01-01")
+    # begin_datetime_str = request.args.get("beginDateTime", "2000-01-01")
```

Comment:
👣 ?

Reply:
should we keep the default though?

Reply:
Yep, I've kept the default, but it's now explicitly on the api endpoint declaration in main.py:149, which means it also shows up in the docs